code/modules/mob/living/carbon/carbon.dm

```diff
@@ -83,11 +83,18 @@
 		mode() // Activate held item
 
 /mob/living/carbon/attackby(obj/item/I, mob/user, params)
-	if(lying && surgeries.len)
-		if(user != src && user.a_intent == INTENT_HELP)
+	var/be_nice = FALSE
+	if(lying && user.a_intent == INTENT_HELP)
+
+		if((I.sharpness || istype(I, TOOL_SCREWDRIVER) || istype(I, /obj/item/coin)) && can_operate(src))//sorry for the snowflake, kids! At least the istype() check won't proc if the item is sharp to start with
+			attempt_initiate_surgery(I, src, user)

```

Comment:
Drapeless surgery? Sounds rad, might be worth mentioning in the changelog tho

Reply:
Oracle removed all drapes pretty early on, so it makes sense that in a direct translation it doesn't have it.

Reply:
never seen drapes used when opening up a toaster with a screwdriver but that's just me